icu4c/source/test/intltest/calcasts.cpp

```diff
@@ -499,5 +501,81 @@ void CalendarCaseTest::Ethiopic() {
+       {2013762.5,  0,   928,         5,        3,  WED,   0,  0,  0}, // Gregorian: 23/05/0801 - not a leap month - not a leap day
+       {2041047.5,  0,  1003,        66,        1,  TUE,   0,  0,  0}, // Gregorian: 04/02/0876 - leap month - not a leap day
+       {2053145.5,  0,  1036,         2,       21,  THU,   0,  0,  0}, // Gregorian: 21/03/0909 - not a leap month - not a leap day
+       {2085852.5,  0,  1125,         9,        9,  SUN,   0,  0,  0}, // Gregorian: 07/10/0998 - not a leap month - not a leap day
+       {2095540.5,  0,  1152,         3,       11,  SUN,   0,  0,  0}, // Gregorian: 17/04/1025 - not a leap month - not a leap day
+       {2150153.5,  0,  1301,         9,       87,  SAT,   0,  0,  0}, // Gregorian: 26/10/1174 - not a leap month - leap day
+       {2176923.5,  0,  1375,         1,        7,  MON,   0,  0,  0}, // Gregorian: 10/02/1248 - not a leap month - not a leap day
+       {2225643.5,  0,  1508,         6,        2,  MON,   0,  0,  0}, // Gregorian: 02/07/1381 - not a leap month - not a leap day
+
+       {2247877.5,  0,  1569,         3,       29,  WED,   0,  0,  0}, // Gregorian: 18/05/1442 - not a leap month - not a leap day - Dharmapala Day
+       {2304268.5,  0,  1723,         8,       17,  TUE,   0,  0,  0}, // Gregorian: 08/10/1596 - not a leap month - not a leap day
+       {2327497.5,  0,  1787,         4,        5,  FRI,   0,  0,  0}, // Gregorian: 14/05/1660 - not a leap month - not a leap day
+       {2372681.5,  0,  1910,        12,        8,  THU,   0,  0,  0}, // Gregorian: 29/01/1784 - not a leap month - not a leap day - Medicine Buddha & Tara Day
+       {2401910.5,  0,  1991,         1,        1,  MON,   0,  0,  0}, // Gregorian: 08/02/1864 - not a leap month - not a leap day - Losar
+       {2436066.5,  0,  2084,         6,       20,  THU,   0,  0,  0}, // Gregorian: 15/08/1957 - not a leap month - not a leap day
+       {2473029.5,  0,  2185,         9,       11,  SUN,   0,  0,  0}, // Gregorian: 27/10/2058 - not a leap month - not a leap day 
+       {2496850.5,  0,  2250,        11,       30,  SUN,   0,  0,  0}, // Gregorian: 16/01/2124 - not a leap month - not a leap day - Shakyamuni Buddha Day New Moon
+       {2524704.5,  0,  2327,         3,        7,  MON,   0,  0,  0}, // Gregorian: 21/04/2200 - not a leap month - not a leap day
+       {2567015.5,  0,  2442,        12,       30,  THU,   0,  0,  0}, // Gregorian: 24/02/2316 - not a leap month - not a leap day - Shakyamuni Buddha Day New Moon
+       { -1,-1,-1,-1,-1,-1,-1,-1,-1 }
+    };
+
+    UErrorCode status = U_ZERO_ERROR;
+    Calendar *c = Calendar::createInstance("bo_IN@calendar=tibetan", status);
+    if (failure(status, "Calendar::createInstance", TRUE)) return;
+    c->setLenient(TRUE);
+    doTestCases(tests1, c);
+    delete c;
+
+
+    //Testing Tsurphu calculations 
```

Comment:
For catching issue issers, could you break the testing against "bo_IN@calendar=tibetan" and "bo_IN@calendar=tibetan-tsurphu" in two different test function. Basically, create another CalendarCaseTest::TibetanTsurphu() and put the following test cases into that part. This may help us to parallelized our test and run faster. Thanks